components/feature/accounts/src/main/java/mozilla/components/feature/accounts/FirefoxAccountsAuthFeature.kt

```diff
@@ -10,19 +10,23 @@ import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 import mozilla.components.concept.engine.EngineSession
 import mozilla.components.concept.engine.request.RequestInterceptor
-import mozilla.components.feature.tabs.TabsUseCases
 import mozilla.components.service.fxa.manager.FxaAccountManager
 import kotlin.coroutines.CoroutineContext
 
 /**
  * Ties together an account manager with a session manager/tabs implementation, facilitating an
  * authentication flow.
+ * @property accountManager .
+ * @property redirectUrl This is the url that will be reached at the final authentication state.
+ * @property coroutineContext The context where [onBeginAuthentication] will be executed,
```

Comment:
`The context which will be used to execute network requests necessary to initiate authentication. Note that [onBeginAuthentication] will be executed using this context.`

Reply:
Fixed!